microsoft/testsuites/cpu/cpusuite.py

```diff
@@ -51,3 +61,69 @@ class CPUSuite(TestSuite):
     )
     def verify_cpu_hot_plug(self, log: Logger, node: Node) -> None:
         verify_cpu_hot_plug(log, node)
+
+    @TestCaseMetadata(
+        description="""
+            This test will check cpu hotplug with storage workload.
+            The cpu hotplug steps are same as `verify_cpu_hot_plug` test case.
+            """,
+        priority=3,
+        requirement=simple_requirement(
+            min_core_count=32,
+        ),
+    )
+    def verify_cpu_offline_storage_workload(self, log: Logger, node: Node) -> None:
+        # run fio process asynchronously on the node and give it 10 seconds to start
+        fio_storage_workload_script = CustomScriptBuilder(
+            Path(__file__).parent.joinpath("scripts"), ["fio_storage_worload.sh"]
+        )
+        process = node.tools[Fio].run_script_async(fio_storage_workload_script)
```

Comment:
When is the process ended?

Reply:
I will add code to explicitly kill the process after we complete the test. The test needs to be run when a storage workload is running.

Reply:
When you add the logic to kill the process, put it in `final`. So, even there is an exception, it can be killed too.